### ISSUE TITLE
[Input]: Clear icon cannot be selected with keyboard

### ISSUE THREAD

**Issue by mhjzhang** [SAP/ui5-webcomponents#10632]
### Bug Description

When the clear icon is enabled for Input, it cannot be selected with tab or any other keys. This could be an accessibility issue.

### Affected Component

Input

### Isolated Example

https://sap.github.io/ui5-webcomponents/components/Input/#clear-icon

### Steps to Reproduce

1. Go to the Input example page above.
2. Type something in the input so the clear icon appears.
3. Press tab, and the focus leaves the input instead of going to the clear icon.

### Priority

Medium

### UI5 Web Components Version

Latest

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by nnaydenow** [comment]
Hi @mhjzhang,

According to the specification, it should not be accessible via the keyboard. It is designed to be accessible with the mouse only, allowing users to perform certain actions with the mouse and without relying on the keyboard.

Additionally, there are several alternatives to the clear icon when the input field is focused:

- CTRL/CMD + A, followed by Del/Backspace
- Del/Backspace directly

Let me know if you need further clarification!
